client/src/utils/Drag.ts

```diff
@@ -220,8 +221,8 @@ const updateDropzones = () => {
     const canDrop = dropTarget ? checkCanDrop(dropTarget, classPeriod) : false;
 
     const isDropTarget =
-      (classPeriod && classPeriod === dropTarget) || // is period, and period is drop darget
-      (!classPeriod && !isScheduledPeriod(dropTarget)); // is inventory, and drop target is inventory class
+      (classPeriod !== null && classPeriod === dropTarget) || // is period, and period is drop darget
+      (classPeriod == null && !isScheduledPeriod(dropTarget)); // is inventory, and drop target is inventory class
```

Comment:
is `classPeriod` always `null`? If so, then:
```suggestion
      (classPeriod !== null && classPeriod === dropTarget) || // is period, and period is drop darget
      (classPeriod === null && !isScheduledPeriod(dropTarget)); // is inventory, and drop target is inventory class
```

Reply:
Resolved [0cffb3](https://github.com/devsoc-unsw/notangles/pull/1040/commits/0cffb32ae0fa20f1fca2329832e471aea77eaa66#diff-b3e2501c0c75a6c116323fbfdef3c5510e3500cebfdbf9a9253f1df0469e8f8cR225)